lua/lspconfig/server_configurations/dafny.lua

```diff
@@ -0,0 +1,20 @@
+local util = require 'lspconfig.util'
+
+local root_files = {
+    'compile_commands.json',
+    '.ccls',
+}
+return {
+  default_config = {
+    filetypes = { 'dfy', 'dafny'},
```

Comment:
```suggestion
    filetypes = { 'dfy', 'dafny' },
```

Reply:
Ok I will do this a bit later today. But I currently don't have a windows machine. Probably I need help for that.